deploy/docker-compose/README.md

```diff
@@ -0,0 +1,390 @@
+
+| Service                                                         | Host IP/Port           | Internal IP/Port         |
+| --------------------------------------------------------------- | ---------------------- | ------------------------ |
+| MinIO (default storage service)                                 | http://localhost:9001  | http://172.29.0.10:9001  |
+| RabbitMQ (default message broker service)                       | http://localhost:15672 | http://172.29.0.20:15672 |
+| MongoDB (default database for Worklflow Manager & Task Manager) | http://localhost:27017 | http://172.29.0.30:27017 |
+| Orthanc UI (optional)                                           | http://localhost:8042  | http://172.29.0.100:8042 |
+| Orthanc SCP (optional)                                          | 4242                   | 4242                     |
+
+Note: Orthanc is included for convenience and to demo end-to-end workflow execution. It may be disabled and have MONAI Deploy Lite integrated with external Orthanc, PACS or other DICOM devices. 
+
+### Common/Known Issues
+
+- The following error indicates that the `docker compose` version may not be up-to-date. If the problem persists, modify the `TASK_MANAGER_DATA` variable defined in the `.env` file and change `$PWD/.md/mdtm` to a fully qualified path. E.g. `/home/monai/some/path/to/.md/mdtm/`.
+	```
+	ERROR: Named volume "$PWD/.md/mdtm:/var/lib/monai:rw" is used in service "task-manager" but no declaration was found in the volumes section. 
+	```
+
+	*WHY? The value of `TASK_MANAGER_DATA` is exposed to the Task Manager container as an environment variable in order for Task Manager to map the volume from the host to the MAP container correctly.*
+
+- The Informatics Gateway service is unhealthy and fails to start with the following error if Docker is not up-to-date:
+  ```bash
+  $ docker logs mdl-ig
+  Failed to create CoreCLR, HRESULT: 0x80070008
+  ```
+  
+## Running a MONAI Deploy Workflow
+
+This package includes Orthanc running and connected to the Informatics Gateway, with all required AE Titles pre-configured.
+
+To get started, download the following DICOM datasets and upload them to Orthanc.
+
+- [Chest CT dataset](https://drive.google.com/file/d/1IGXUgZ7NQCwsix57cdSgr-iYErevqETO/view?usp=sharing)
+- [Abdomen CT dataset](https://drive.google.com/file/d/1d8Scm3q-kHTqr_-KfnXH0rPnCgKld2Iy/view?usp=sharing)
+
+Note: these DICOM datasets were converted to DICOM from Medical Decathlon training and validation mages & may be used for the referenced examples later in this document.
+
+### Upload DICOM Datasets
+
+Navigate to Orthanc via http://localhost:8042, and click *Upload* on the top right-hand corner.
+Drag & drop any DICOM files (no folders) to the blue box on the page and then click *Start the upload*.
+
+Or, upload your files to Orthanc using the *storescu* command from [dcmtk](https://dcmtk.org/dcmtk.php.en).
+```
+storescu -v +r +sd -aec ORTHANC localhost 4242 /path/to/my/dicom/*
+```
```

Comment:
Here it is not clear if the .zip files are uploaded to Orthanic, or users need to unzip the linked data files above first.

Reply:
updated section